backend/keystore/keystore.go

```diff
@@ -123,7 +124,7 @@ type Keystore interface {
 	// SignBTCMessage signs the message using the private key at the keypath. The scriptType is
 	// required to compute and verify the address. The returned signature is a 65 byte signature in
 	// Electrum format.
-	SignBTCMessage(message []byte, keypath signing.AbsoluteKeypath, scriptType signing.ScriptType) ([]byte, error)
+	SignBTCMessage(message []byte, keypath signing.AbsoluteKeypath, scriptType signing.ScriptType, coin messages.BTCCoin) ([]byte, error)
```

Comment:
that's the wrong parameter, as it is bitbox02 specific, and bitbox02 is only one implementation of the keystore. the right param is coin.Code, like e.g. in CanSignMessage above.